deploy/install/02-components/04-driver.yaml

```diff
@@ -38,6 +38,12 @@ spec:
           # manually set root directory for csi
           #- --kubelet-root-dir
           #- /var/lib/rancher/k3s/agent/kubelet
+          # manually specify number of CSI attacher replicas
+          #- --csi-attacher-replica-count
+          #- "3"
```

Comment:
Wrong type.

Reply:
Without quotation marks, `Error from server (BadRequest): error when creating "deploy/install/02-components/04-driver.yaml": Deployment in version "v1beta2" cannot be handled as a Deployment: v1beta2.Deployment.Spec: v1beta2.DeploymentSpec.Template: v1.PodTemplateSpec.Spec: v1.PodSpec.Containers: []v1.Container: v1.Container.Env: []v1.EnvVar: Command: []string: ReadString: expects " or n, but found 4, error found in #10 byte of ...|a-count",4,"--csi-pr|..., bigger context ...|-backend:9500/v1","--csi-attacher-replica-count",4,"--csi-provisioner-replica-count",5],"env":[{"nam|...` occurred.

Reply:
Oh, it was treated as a string always.